### PR TITLE
Update from auto-update/renku-ui-0.10.2

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request to update from branch auto-update/renku-ui-0.10.2

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+There are several acceptance tests, which are all in the ch.renku.acceptancetests package.
+
+| Class Name                | Purpose                                                          |
+| ------------------------- | ---------------------------------------------------------------- |
+| LoginSpec                 | Just check that login/logout works                               |
+| MinimalFunctionalitySpec  | Check that the basic features are working                        |
+| HandsOnSpec               | Run through the hands-on tutorial                                |
+| ExistingProjectSpec       | Not normally run, but useful for debugging/developing            |
+
+Some tests create new projects (MinimalFunctionalitySpec, HandsOnSpec). In these cases, the project name is normally of the form `test_[timestamp]`. This makes it easy to identify the test projects and generally avoid name collisions. If the test runs successfully to completion, the project is deleted at the end of the test. If the test fails, then the project is left around to aid debugging.
```
Comment (rokroskar) [inline tests/README.md:14]: could you please wrap this at some reasonable character count?

Comment (ciyer) [reply tests/README.md:14]: It's markdown. What is the advantage of hard wrapping the content?

Comment (rokroskar) [reply tests/README.md:14]: it's a text file. single-line text files are annoying to read.

**Comment by rokroskar** [comment]
I squashed the doc commits - will do a rebase merge.

**Pull request merged**